packages/meditrak-app-server/src/__tests__/__integration__/sync/PushChangesRoute.test.ts

```diff
@@ -363,26 +363,6 @@ describe('changes (POST)', () => {
     });
 
     describe('Reject if invalid', () => {
-      it('rejects if answer provide non-exist question_id', async () => {
```

Comment:
This test is no longer enforced. At the meditrak-app-server level, if a survey response comes in with a question_id that doesn't exist we'll still pass it through to the central-server.

At create time this will fail due to foreign key constraint. All survey responses are created within a transaction so no responses will be saved.

I think I'm okay with this. But happy to add a piece of validation to check all the questions exist.